charts/seed-monitoring/charts/grafana/dashboards/owners/controlplane-logs-dashboard.json

```diff
@@ -349,49 +372,63 @@
     "templating": {
       "list": [
         {
-          "allValue": "",
+          "allValue": ".+",
           "current": {
             "selected": true,
-            "text": "{{ (index . 0).PodPrefix }}",
-            "value": "{{ (index . 0).PodPrefix }}"
+            "tags": [],
+            "text": [
+              "kube-apiserver-5479b9f94f-b246w"
+            ],
+            "value": [
+              "kube-apiserver-5479b9f94f-b246w"
```

Comment:
I'm not sure if there is a better way to avoid persisting this initial value in the dashboard definition.